e2e-tests/src/pages/demo/home-about-us.ts

```diff
@@ -0,0 +1,25 @@
+import { expect, Locator, Page } from '@playwright/test'
+import data from 'src/data/ourhealth-en.json'
```

Comment:
should this reference demo-specific data instead of crossing over into ourhealth? is there anything that isn't study-specific that we can pull out into a common data file? (same comment applies to `home.ts` as well)

Reply:
I suspect there's more work to do here - I think we should step back and look at it as a team and figure out how we want to approach making the common/abstract classes we have work for every study, especially in regards to multi-language testing. I suspect it will be a decent amount of work, my thought was that I would do the minimum to make it work with demo and we all get together to think about our approach, but I'm happy to move this json over in this ticket if y'all think it would be useful.

Reply:
Added this to eng strategy meeting discussion - I think a spike where someone focuses on upgrading the abstract classes / figuring out multi-lang stuff is probably the right approach.